README.md

```diff
@@ -14,11 +14,45 @@ dependencies:
 **2:** Install packages from the command line:
 
 ```bash
-$ flutter packages get
+flutter packages get
 ```
 
 Alternatively, your editor might support flutter packages get. Check the docs for your editor to learn more.
 
+## Configuration
+
+For using background_location package you need to add permissions to use location service. **Add** these permission to your platform.
+
+### iOS Platform Permission
+
+in `ios/Runner/Info.plist` add:
+
+```xml
+<key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
+<string>This app needs access to location.</string>
```

Comment:
these location strings are pretty generic and may get apps held up by apple in app review. Could you maybe add a note about customizing them? or maybe add a placeholder to suggest that the user should fill in their own content?

Reply:
I agree, i think use that string in my project too😅  i'll add note for that.

Reply:
Added note to ios configuration in this [commit](https://github.com/Almoullim/background_location/pull/226/commits/83b9bf4dd1ebf2ca8188d731335a54eb1815688b)

Reply:
i feel like the language should probably be stronger. IMO i think putting an obvious placeholder like `[Reason why your app needs location]` in the string itself and possibly also having a description saying apple may reject apps for incorrect or incomplete descriptions would be good.

Reply:
Ohh i see, commit on the way

Reply:
Okey done